### PR TITLE
chore: update consentRequests response structure

### PR DESCRIPTION
**Pull request by sridharvoruganti**

### CODE REVIEW

File: src/simulator/api.yaml [hunk 1]
```diff
@@ -2124,15 +2124,20 @@ components:
...
-          items:
-            $ref: '#/components/schemas/ConsentRequestChannelType'
-        authUri:
-          type: string
-          description: >-
-            The callback uri that the pisp app redirects to for user to complete
-            their login.
+        data:
+          type: object
+          properties:
```
Comment (eoln) [inline src/simulator/api.yaml:2129]: are `authChannles` and `authUri` are not required?

File: src/simulator/api.yaml [hunk 2]
```diff
@@ -2124,15 +2124,20 @@ components:
...
-        authChannels:
-          type: array
-          items:
-            $ref: '#/components/schemas/ConsentRequestChannelType'
-        authUri:
-          type: string
-          description: >-
-            The callback uri that the pisp app redirects to for user to complete
-            their login.
+        data:
```
Comment (eoln) [inline src/simulator/api.yaml:2127]: `data` is very generic name, what about `validateResponse` ?

File: src/simulator/api.yaml (hunk 1 above)
Comment (sridharvoruganti) [reply src/simulator/api.yaml:2129]: generally we are not specifying any mandatory fields in simulator API specification, response is based on rules

File: src/simulator/api.yaml (hunk 2 above)
Comment (sridharvoruganti) [reply src/simulator/api.yaml:2127]: I think data is fine, isValid is validation response and data contains some extra information returned by dfsp

**Review by eoln — APPROVED** [review]

**Pull request merged**